### DISCUSSION TITLE
routing via corridor

### DISCUSSION THREAD

**Discussion by APneunzehn74** [osmandapp/OsmAnd#15470] [Maps Related]
Hello I noticed today that the OsmAnd bike router routes via highway=corridor.  This is not correct as bicycles are not allowed here.  Basically, I also noticed that people like to route via sidwalk, which is not okay without explicit bike approval.
Greetings Andreas

**Comment by vshcherb** [comment]
The rules to be reviewed are located in configuration https://github.com/osmandapp/OsmAnd-resources/blob/master/routing/routing.xml#L1039
